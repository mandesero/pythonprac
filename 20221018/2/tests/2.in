:x 1
x
:foo x y z w x+y+z+w
foo 1 2 3 4
quit "calls : {} lines : {}"
