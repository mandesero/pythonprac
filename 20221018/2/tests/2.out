1
10
calls : 3 lines : 5
